src/protagonist/API.Client/DlcsClient.cs

```diff
@@ -317,7 +317,38 @@ public async Task<CustomerQueue> GetQueue()
         var queue = await response.ReadAsHydraResponseAsync<CustomerQueue>(jsonSerializerSettings);
         return queue;  
     }
+    
+    public async Task<IEnumerable<NamedQuery>> GetNamedQueries(bool includeGlobal)
+    {
+        var url = $"customers/{currentUser.GetCustomerId()}/namedQueries";
+        var response = await httpClient.GetAsync(url);
+        var namedQueries = await response.ReadAsHydraResponseAsync<HydraCollection<NamedQuery>>(jsonSerializerSettings);
+
+        return includeGlobal
+            ? namedQueries.Members
+            : namedQueries.Members.Where(nq => nq.Global == false);
+    }
+
+    public async Task DeleteNamedQuery(string namedQueryId)
+    {
+        var url = $"customers/{currentUser.GetCustomerId()}/namedQueries/{namedQueryId}";
+        await httpClient.DeleteAsync(url);
+    }
 
+    public async Task UpdateNamedQuery(string namedQueryId, string template)
+    {
+        var url = $"customers/{currentUser.GetCustomerId()}/namedQueries/{namedQueryId}";
+        await httpClient.PutAsync(url, ApiBody(new NamedQuery(){ Template = template }));
+    }
+    
+    public async Task<NamedQuery> CreateNamedQuery(NamedQuery newNamedQuery)
+    {
+        var url = $"customers/{currentUser.GetCustomerId()}/namedQueries";
+        var response = await httpClient.PostAsync(url, ApiBody(newNamedQuery));
+        var namedQuery = await response.ReadAsHydraResponseAsync<NamedQuery>(jsonSerializerSettings);

```

Comment:
How would this be represented in UI if it wasn't a `NamedQuery` (e.g. it failed and you got an `application/problem+json` back?